python/tests/test_examples.py

```diff
@@ -368,6 +368,7 @@ def test_char_vs_string(self):
         self.assertEqual(repr(metta.run('!("A")')), '[[("A")]]')
         self.assertEqualMettaRunnerResults(metta.run("!(get-type 'A')"), [[S('Char')]])
         self.assertEqualMettaRunnerResults(metta.run('!(get-type "A")'), [[S('String')]])
+        self.assertEqual(metta.run('!(repr "A")')[0][0].get_object(), ValueObject("\"A\""))
```

Comment:
Please move test into `test_grounded_type.py` as this is a change in a grounded atom's behaviour.